timesketch/lib/graphs/interface.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Interface for graphs."""
+
+import hashlib
+
+from flask import current_app
+import networkx as nx
+
+from timesketch.lib.datastores.elastic import ElasticsearchDataStore
+
+
+GRAPH_TYPES = {
+    'Graph': nx.Graph,
+    'MultiGraph': nx.MultiGraph,
+    'DiGraph': nx.DiGraph,
+    'MultiDiGraph': nx.MultiDiGraph
+}
+
+
+class Graph(object):
```

Comment:
no need to add the `(object)`, we are no longer writing code that is both py2/py3